kibbeh/src/modules/room/chat/RoomChatList.tsx

```diff
@@ -43,6 +45,16 @@ export const RoomChatList: React.FC<ChatListProps> = ({ room }) => {
     estimateSize: React.useCallback(() => 20, []),
   });
 
+  const getBadgeIcon = (m: Message) => {
+    const user = users.find((u) => u.id === m.userId);
+    const isSpeaker = room.creatorId === user?.id || user?.roomPermissions?.isSpeaker;
+    let emoji = null;
+    if (isSpeaker) {
+      emoji = "📣";
+    }
+    return emoji && <Twemoji text={emoji} style={{ marginRight: "1ch" }}/>;
```

Comment:
can you add a title to it so people can see what it means on hover

Reply:
Done